tutorials/hello/hello_client/test/unit/modules/hello/commands/__snapshots__/create_hello_command.spec.ts.snap

```diff
@@ -0,0 +1,20 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`CreateHelloCommand constructor should have valid schema 1`] = `
+{
+  "$id": "hello/createHello-params",
+  "properties": {
+    "message": {
+      "dataType": "string",
+      "fieldNumber": 1,
+      "maxLength": undefined,
+      "minLength": undefined,
```

Comment:
maybe its better to put a limit here like min 1 and max 256, if in case you don't want to put a limit then remove these 2 props entirely

Reply:
```suggestion
      "maxLength": 1,
      "minLength": 256,
```